docs/extensibility/extending-blocks.md

```diff
@@ -0,0 +1,127 @@
+# Extending Blocks (Experimental)
+
+## Modifying Blocks
+
+To modify the behaviour of existing blocks, Gutenberg exposes the following filters:
```

Comment:
A lot of persons do ask about the hooks and filters we should link to the hooks package README or something for clarity.

Reply:
Oh yes, good idea. 👍